src/languages/en.js

```diff
@@ -216,6 +216,12 @@ export default {
     supportTelegramChatID: "Support Direct Chat / Group / Channel's Chat ID",
     wayToGetTelegramChatID: "You can get your chat ID by sending a message to the bot and going to this URL to view the chat_id:",
     "YOUR BOT TOKEN HERE": "YOUR BOT TOKEN HERE",
+    "Silent Notification": "Silent Notification",
+    "Silent Notification Description": "If enabled, the notification in Telegram will be without sound.",
+    "Protect Forwarding": "Protect Forwarding",
+    "Protect Forwarding Description": "If enabled, the notification in Telegram will be protected from forwarding and saving.",
```

Comment:
Maybe change it to
"Protect Forwarding Description": "If enabled, the notification in Telegram will be protected from forwarding and saving.",